packages/label-sync/src/labels.json

```diff
@@ -55,6 +55,11 @@
       "description": "Request for information or clarification. Not an issue.",
       "color": "c5def5"
     },
+    {
+      "name": "priority: p3",
+      "description": "",
```

Comment:
I may suggest that we remove the 'highest, medium, lowest' from others, and update descriptions a bit. For this one, perhaps "Desirable enhancement or fix. May not be include in next release"